### DISCUSSION TITLE
Welcome to course-api-framework-python Discussions!

### DISCUSSION THREAD

**Discussion by automationhacks** [automationhacks/course-api-framework-python#1] [General]
## 👋 Welcome!

  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by nfredrik** [comment]
Hi!
Fredrik: work as developer and interested in testing. I worked with both cucumber/ruby and behave/python and testing rest-apis.

I spotted this course and it's really good! Thanks!  I agree on much said/written.  I had some issues on some of the code, but I 
can give feedback on that later.

One thing to dig deeper into would be: idempotency, see
https://nordicapis.com/understanding-idempotency-and-safety-in-api-design/

I guess it's nice verify that your SUT stick to this behaviour.

Thanks a lot!

**Reply by automationhacks** [reply]
Thanks for your comment and i'm glad you liked the course. Feel free to raise issues in case you found some issues and i'll take a look at it.

**Comment by laxmikantm** [comment]
Hi @automationhacks ,

Nice work & contribution to tech community.
One question - how to use this framework to test URL based public API endpoints?
Thanks

**Reply by automationhacks** [reply]
Thanks, you just need to tweak the base url in the endpoint for your use case.
